package.json

```diff
@@ -23,29 +23,33 @@
   "bugs": {
     "url": "https://github.com/bryanpizzillo/r4rimporter/issues"
   },
+  "engines": {
+    "node": "^16"
```

Comment:
Can you do >=16 and <17, as well as set the npm version. You can see this in the cgov-react-app-playground.

Also you need a .npmrc file with the following to get npm to actually enforce the node/npm version. Otherwise npm will just warn.
```
engine-strict=true
```